mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
-  return Math.random() * 10;
+    return Math.random() * 10;
 }
+// Get a random number multiplied by 10
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+    return word1.concat(word2);
 }
 
+// This one is concatenating word1 to word2
+
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+    return firstWord.concat(secondWord, thirdWord);
```

Comment:
there's something small missing here which is why the tests aren't passing. Remember when you join words you need spaces between them.

Reply:
As I said last time, it was just because I did a mistake about the branches and the correct file is in another branch and I couldn't merge them